salt/orchestrate/aws/mitx_elb.sls

```diff
@@ -71,4 +73,64 @@ register_edx_{{ purpose_name }}_nodes_with_elb:
         {% endfor %}
     - require:
         - boto_elb: create_elb_for_edx_{{ purpose_name }}
+
+{% if edx_type == 'live' %}
+{% set elb_name = 'edx-studio-live-{env}'.format(
+   purpose=purpose_name, env=ENVIRONMENT)[:32].strip('-') %}
+create_elb_for_edx_studio_live:
```

Comment:
Wouldn't it be better if this was templatized similar to the previous call?

Reply:
We can't use the same template value because it would conflict with the other instance. In this case the purpose is studio-live but we don't want to introduce that concept everywhere.